bundles/core/pom.xml

```diff
@@ -71,6 +72,7 @@
                             com.adobe.cq.commerce.*;resolution:=optional,
                             com.adobe.aem.wcm.seo.*;resolution:=optional,
                             javax.annotation;version=0.0.0,
+                            javax.annotation.meta;resolution:=optional,
```

Comment:
Is this used?

Reply:
No, only `javax.annotation.*` is used in this repo and only `javax.annotation.PostConstruct` is used in this PR.